### PR TITLE
Fix bug in tutorial step-46

### PR DESCRIPTION
**Pull request by starki0815**
This fixes a bug in tutorial step-46, which caused the y-displacements (z in 3d) to be unsymmetric. As the results are affected, the plots in the documentation maybe need adjustment too ... but I don't know how to do this.

### CODE REVIEW

File: examples/step-46/step-46.cc [hunk 1]
```diff
@@ -798,8 +799,11 @@ namespace Step46
           stokes_fe_face_values.normal_vector(q);
 
         for (unsigned int k = 0; k < stokes_fe_face_values.dofs_per_cell; ++k)
-          stokes_symgrad_phi_u[k] =
-            stokes_fe_face_values[velocities].symmetric_gradient(k, q);
+          {
+            stokes_symgrad_phi_u[k] =
+              stokes_fe_face_values[velocities].symmetric_gradient(k, q);
+            stokes_phi_p[k] = stokes_fe_face_values[pressure].value(k, q);
```
Comment (bangerth) [inline examples/step-46/step-46.cc]: Ugh, did we never initialize this and always just use zero?

**Comment by bangerth** [comment]
/rebuild

Nice work. Does this affect the output from this program as shown in the "Results" section.

File: examples/step-46/step-46.cc (hunk 1 above)
Comment (starki0815) [reply examples/step-46/step-46.cc]: ... even worse. It was passed in non-zero, because the vector was used outside the function.

**Comment by starki0815** [comment]
I think it should affect the displacement plots, although it's probably not visible because the differences are not too large.

**Comment by starki0815** [comment]
What I am also just started wondering about: In the equilibrium equation in the tutorial (the very first equation), the total stress is obviously `(2*eta*epsilon - p*I)` - but in the interface condition the traction is `(2*eta*epsilon + p*I)*n`. Is this really what is intended [Integration by parts of the first two terms of the weak form also gives a `(2*eta*epsilon - p*I)*n` term, rather than `(2*eta*epsilon + p*I)*n`] ?

**Comment by bangerth** [comment]
Probably not :-)

**Comment by starki0815** [comment]
Ok ... I'll double check (I tend to think that there should be a `-` in front of the `eta` for the equations to make) and try to fix it if it turns out that I am right.

**Comment by starki0815** [comment]
It really seems to be wrong. The way the solid is deforming in 2d doesn't make sense. I have just for testing set the pressure contribution to the interface traction to zero and simultaneously applied the traction resulting from the velocity gradient exclusively on the top face of the pillar. In this case, the pillar should definitely bend to the right, which it doesn't do in the simulation. With a minus in front of the `eta` in the interface traction term all is fine. And the theory does also say that there must be a minus.

I can fix the documentation. But how to proceed with the results section (things will really change there)? I guess the plots have been done in visit, which I am not using at the moment. Also, I guess that the last refinement step of the 3d simulations will run out of memory on my machine. @bangerth According to the documentation, you have contributed the example. Can you do the results if I fix the rest?

**Comment by starki0815** [comment]
I have adjusted the interface traction term and updated the documentation accordingly. Only the 2d-plots and the 3d refinement results and plots need still to be done. Unfortunately, I can't even get visit to work on my system right now and the 3d computation runs out of memory as expected. So, I can't do these things.

**Comment by bangerth** [comment]
I can do this, but it will have to wait until after the ASPECT hackathon. So in June. Thanks for figuring all of this out!

**Comment by masterleinad** [comment]
It should be fine to fix the bug and update the plots in a second step.

/rebuild

**Comment by bangerth** [comment]
Oh, this totally fell of my radar :-( I really apologize for letting this drag on for this long!

**Pull request merged**